rpm.mk

```
@@ -26,6 +26,19 @@ RPMBUILD_OPTIONS += $(if $(filter 1, $(BUNDLE_LIBDB)),--with bundle_libdb,--with
 LIBDB_URL ?= $(shell rpmspec $(RPMBUILD_OPTIONS) -P $(RPMBUILD)/SPECS/389-ds-base.spec | awk '/^Source4:/ {print $$2}')
 LIBDB_TARBALL ?= $(shell basename "$(LIBDB_URL)")
 
+# Enable BUNDLE_BDBREADERS if neither BUNDLE_LIBDB nor /usr/include/db.h is available
+ifeq ($(BUNDLE_LIBDB), 1)
+BUNDLE_BDBREADERS ?= 0
+else
+ifeq (,$(wildcard /usr/include/db.h#))
+BUNDLE_BDBREADERS ?= 1
+else
+BUNDLE_BDBREADERS ?= 0
+endif
+endif
+RPMBUILD_OPTIONS += $(if $(filter 1, $(BUNDLE_BDBREADERS)),--with libbdb-ro,--without libbdb-ro)
```

```suggestion
RPMBUILD_OPTIONS += $(if $(filter 1, $(BUNDLE_BDBREADERS)),--with libbdb_ro,--without libbdb_ro)
```

Even though `libbdb_ro` and `libbdb-ro` expand to the same string, it would be nice to have these strings consistent across the spec file and rpm.mk.